packages/frontend/src/components/CollectionCard/CollectionCard.jsx

```diff
@@ -51,43 +54,43 @@ export default function CollectionCard({ collection }) {
   };
 
   return (
-    <div className="collection-card" css="position: relative">
-      <div className="custom-column left">
-        <h3>
-          <Link to={`/collection/${collection.id}`}>{collection.name}</Link>
-        </h3>
-        <div className="collection-stats">
-          <span>
-            {pluralize('dataset', collection.datasetIds?.length ?? 0, true)}
-          </span>
+    <>
+      <Link to={`/collection/${collection.id}`} className="collection-card">
```

Comment:
All of these line changes are due to a change in indentation. The real change is just changing this `<div>` to a `<Link>`